Libraries/MiscDrivers/Camera/mipi_camera.h

```diff
@@ -35,8 +35,9 @@
 #define LIBRARIES_MISCDRIVERS_CAMERA_MIPI_CAMERA_H_
 
 #include <stdint.h>
+#ifndef OV5640_DVP
 #include "csi2.h"
```

Comment:
In general I think it would be cleanest not to have to check for `OV5640_DVP` in this file.  Even better would be if we could use camera.h when `OV5640_DVP` is selected.  It would require significant restructuring to consolidate camera format structs between the `mipi_camera.h` and `camera.h` though.  

As we continue to add more camera drivers to the SDK I think it would be worth it to spend some time making this process cleaner.  I will put together a proposal/PR for review from all the camera developers